NewDawn/NewDawn/ViewControllers/SettingPageViewControllers/EditProfileViewController.swift

```diff
@@ -67,12 +67,47 @@ class EditProfileTabelViewController: UITableViewController{
     @objc func back(sender: UIBarButtonItem) {
         // Get information from local and send to server
         // Go back to the root ViewController
-        self.dismiss(animated: true, completion: {})
-        self.navigationController?.popViewController(animated: true)
+        let request = EditProfileUtil.createRegistrationRequest()
+        
+        if request == nil {
+            return
+        }
+        let activityIndicator = self.prepareActivityIndicator()
+        DispatchQueue.main.async {
+            self.processSessionTasks(request: request!){
+                register_response in
+                if let images = ImageUtil.getPersonalImagesWithData(), images.count != 0{
+                    var images_count = 0
+                    for single_image in images{
+                        let single_img = single_image["img"]
+                        let single_params = [
+                            "order": single_image["order"]!,
```

Comment:
Can we check if these fields exist before unwrapping them? If any field is missing, we can just continue the for loop

Reply:
Yep. They are default with UNKNOWN value.